docs/docs/FAQ.md

```diff
@@ -29,14 +33,19 @@ As an example, the following modification of ```docker-compose.yml``` will set t
       - TZ=Europe/Stockholm # <---- Add this line in the microservices config
 ```
 
-### Why doesn't Immich watch an existing photo gallery directory?
+### Why are only photos and not videos being uploaded to Immich?
+This often happens when using a reverse proxy or cloudflare tunnel in front of Immich. Make sure to set your reverse proxy to allow large POST requests. In `nginx`, set `client_max_body_size 50000M;` or similar. Cloudflare tunnels are limited to 100 mb file sizes.
 
-The initial approach of Immich is to become a backup tool, primarily for mobile device usage. Thus, all the assets must be uploaded from the mobile client. The app was architectured to perform that job well.
+### Why is Immich slow on low-memory systems like the Raspberry Pi?
+Immich uses optional machine-learning features to enhance search results. This feature, however, can be too heavy to run on a Raspberry Pi. To disable machine learning, comment out the `immich-machine-learning` section of your docker-compose.yml and set `IMMICH_MACHINE_LEARNING_URL=false` in your .env file.
 
 ### What happens to existing files after I choose a new [Storage Template](/docs/administration/storage-template.mdx)?
 
 Template changes will only apply to new assets. To retroactively apply the template to previously uploaded assets, run the Storage Migration Job, available on the [Jobs](/docs/administration/jobs.md) page.
 
+### In the uploads folder, why are photos stored in the wrong date?
```

Comment:
Hello, can you elaborate on this? For the current implementation, this should not happen

Reply:
I think that on file upload the asset.fileCreatedAt defaults to the file created date and that's what is used in the storage template. Then after exif extraction saves the corrected date, the new storage template re-applied with the now changed date.